src/scene/export.ts

```diff
@@ -114,24 +115,37 @@ export const exportToSvg = async (
   if (appState.exportWithDarkMode) {
     svgRoot.setAttribute("filter", THEME_FILTER);
   }
+  let assetPath =
+    window.EXCALIDRAW_ASSET_PATH ||
+    `https://unpkg.com/${pkg.name}@${pkg.version}/dist/excalidraw-assets/`;
+
+  if (assetPath?.startsWith("/")) {
+    const origin = isProdEnv()
+      ? window.location.origin
+      : "https://excalidraw.com";
+    assetPath = assetPath.replace("/", `${origin}/`);
+  }
 
+  const defs = document.createElement("defs");
+  const style = document.createElement("style");
+  style.appendChild(
+    document.createTextNode(
+      "@font-face {" +
+        "font-family: 'Virgil';" +
+        `src: url(${assetPath}Virgil.woff2);` +
+        "}" +
+        "@font-face {" +
+        "font-family: 'Cascadia';" +
+        `src: url(${assetPath}Cascadia.woff2);` +
+        "}",
+    ),
+  );
+  defs.appendChild(style);
```

Comment:
why not keep it inlined as previously, which is cleaner?

Reply:
The inlining will not work since I need to generate dynamic URLs hence injecting the style tag with url

Reply:
all you're doing in injecting the variable there? You could do that same in the SVG string.